sklearn/neighbors/_binary_tree.pxi

```diff
@@ -878,7 +794,7 @@ def newObj(obj):
 
 ######################################################################
 # define the reverse mapping of VALID_METRICS
-from ._dist_metrics import get_valid_metric_ids
+from sklearn.metrics._dist_metrics import get_valid_metric_ids
```

Comment:
Cannot we keep a relative import?

```suggestion
from ..metrics._dist_metrics import get_valid_metric_ids
```

Reply:
Good spot! This was automagically renamed when refactoring.

I guess this import can be moved with the others at the top, L.154.